test/simulator/timestamp/src/include/timestamp_manager.h

```diff
@@ -41,7 +41,7 @@ class timestamp_manager {
     void parse_config(const std::string &, std::map<std::string, std::string> &);
     uint64_t hex_to_decimal(const std::string &);
     std::string decimal_to_hex(const uint64_t);
-    int validate_hex_value(const std::string &);
+    int validate_hex_value(const std::string &, const std::string);
```

Comment:
Should the second arg be passed by reference too?

Reply:
Updated in [bb40f95](https://github.com/wiredtiger/wiredtiger/pull/8372/commits/bb40f9569bf6accd229f186cac8b1aa9a776605d)